examples/gallery/maps/choropleth_map.py

```diff
@@ -14,45 +14,28 @@
 """
 
 # %%
-import geodatasets
 import geopandas as gpd
 import pygmt
 
-# Read the example dataset provided by geodatasets.
-gdf = gpd.read_file(geodatasets.get_path("geoda airbnb"))
-print(gdf.head())
+provider = "https://naciscdn.org/naturalearth"
+world = gpd.read_file(f"{provider}/50m/cultural/ne_50m_admin_0_countries.zip")
```

Comment:
Any specific reason for using the 50m dataset? The 110m dataset also works and is smaller.
```suggestion
world = gpd.read_file(f"{provider}/110m/cultural/ne_110m_admin_0_countries.zip")
```

Reply:
No, I think we started with 50 m and I actually did not test if 110 m would be enough.